### DISCUSSION TITLE
连接WIFI的平板该怎么设置？

### DISCUSSION THREAD

**Discussion by windstander** [Mygod/VPNHotspot#361] [Q&A]
我有一个连接WIFI的平板，没有4G或者5G功能，向创建带VPN的热点给电视机用。
我用这个APP，是该点开WLAN热点再点开生成的WLAN1，还是直接点开“无线中继”就可以了。
![1611134121839](https://user-images.githubusercontent.com/55864125/162391999-0d9bc473-06be-4a7a-a2f3-0f40aae7aa81.jpg)
@Mygod

**Comment by windstander** [comment]
试了，应该开无线中继，不过Nexus9平板开了无线中继，生成的热点无法上网。

**Reply by Mygod** [reply]
Either should be fine.

**Reply by windstander** [reply]
> Either should be fine.

测试，Nexus9只能开无线中继，开了的热点不能上网。不是官方固件，是xda的lingeOS，下次我导出日志给你看看。

**Reply by windstander** [reply]
> Either should be fine.
[vpnhotspot-745090767051009702.log](https://github.com/Mygod/VPNHotspot/files/8479702/vpnhotspot-745090767051009702.log)
现在中转都打不开了。

**Reply by windstander** [reply]
@Mygod 帮我看看日志呢？

**Reply by Mygod** [reply]
Did any previous version work for you?

**Reply by Mygod** [reply]
Any error you see in the UI?

**Reply by windstander** [reply]
> 

I never used previous versions.

**Reply by windstander** [reply]
> Any error you see in the UI?

NO, now my nexus 9 system is:
https://forum.xda-developers.com/t/unofficial-lineageos-15-1-for-nexus-9-wifi-feb-24-2022.4244361/

I used this system before, the repeater can work, but without internet connection.
https://forum.xda-developers.com/t/unofficial-8-1-0-lineageos-15-1-for-nexus-9-wifi-07-25-2020.4051433/

**Reply by Mygod** [reply]
Your ROM might be bugged. Report to ROM developer instead.
